src/textual/app.py

```diff
@@ -566,22 +567,32 @@ def export_screenshot(self, *, title: str | None = None) -> str:
         console.print(screen_render)
         return console.export_svg(title=title or self.title)
 
-    def save_screenshot(self, path: str | None = None) -> str:
-        """Save a screenshot of the current screen.
+    def save_screenshot(
```

Comment:
Could note this down to mention as a HOWTO.

Reply:
Saving screenshots? Feels a bit too trivial for a how-to, as long as it appears in the search...